### PR TITLE
[14.0] mig product_supplierinfo_for_customer_invoice

### PR DESCRIPTION
**Pull request by fredzamoabg**
**product_supplierinfo_for_customer_invoice** module migration from 12.0
depends on product_supplierinfo_for_customer module migration https://github.com/OCA/product-attribute/pull/859

### CODE REVIEW

File: product_supplierinfo_for_customer_invoice/models/account_invoice.py [hunk 1]
```diff
@@ -2,31 +2,28 @@
...
-            code_id = product_customerinfo_obj.search(
-                [
-                    ("product_tmpl_id", "=", product.product_tmpl_id.id),
-                    ("name", "=", line.invoice_id.partner_id.id),
-                ],
-                limit=1,
-            )
+        for line in self:
+            code_id = product_customerinfo_obj.browse()
+            if line.move_id.move_type in ["out_invoice", "out_refund"]:
```
Comment (SimoneVagile) [inline product_supplierinfo_for_customer_invoice/models/account_invoice.py]: You can use [this](https://github.com/odoo/odoo/blob/14.0/addons/account/models/account_move.py#L2043) here ;)

**Review by SimoneVagile — CHANGES_REQUESTED** [review]

**Comment by SimoneVagile** [comment]
@fredzamoabg Please check Travis and runbot :red_circle:

**Comment by fredzamoabg** [comment]
> @fredzamoabg Please check Travis and runbot

of course depends on https://github.com/OCA/product-attribute/pull/859 as described

File: product_supplierinfo_for_customer_invoice/models/account_invoice.py (hunk 1 above)
Comment (fredzamoabg) [reply product_supplierinfo_for_customer_invoice/models/account_invoice.py]: done, thanks

**Review by SimoneVagile — APPROVED** [review]

**Comment by SimoneVagile** [comment]
Code review LGTM.

**Pull request closed without merging**